torch_brain/transforms/unit_filter.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable
+
+import numpy as np
+
+from temporaldata import ArrayDict, Data, IrregularTimeSeries, RegularTimeSeries
+
+
+class FilterUnit:
+    r"""
+    Drop units based on the `filter_fn` given in the constructor.
+    This operation is performed in-place.
+    """
+
+    def __init__(
+        self,
+        filter_fn: Callable[[np.ndarray], np.ndarray],
```

Comment:
`filter_fn` -> `mask_fn` because the function gives you a mask, and doesn't do the filtering itself.

Reply:
Also the input to the fn is a `ArrayDict`, so the type definition needs to be fixed.